provisioning/roles/kuma/tasks/python.yml

```diff
@@ -2,17 +2,24 @@
 - name: Install virtualenv
   pip: name=virtualenv
 
-- name: Install Python packages in virtualenv
+- name: Install pip
   pip:
-    executable: "/home/vagrant/env/bin/python /home/vagrant/src/scripts/peep.py"
-    requirements: "{{ item }}"
-    virtualenv: "/home/vagrant/env"
+    name: pip
+    version: 8.0.0
+    virtualenv: /home/vagrant/env
+
+- name: Install local dependencies in virtualenv
+  pip:
+    requirements: /home/vagrant/src/requirements/local.txt
+    virtualenv: /home/vagrant/env
     virtualenv_site_packages: yes
   with_items:
```

Comment:
The way this got changed, this `with_items` has no value, so it breaks vagrant provisioning.